package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "csharp",
   "publisher": "ms-vscode",
-  "version": "1.0.11",
+  "version": "1.0.12",
```

Comment:
Suggestion: use version 1.1.0 to signify that we have a bigger chunk of changes. Also we still might want to ship a '1.0.12' with some low-risk-but-important fix between now and when we get the next version out.